.mergify.yml

```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: automatically merge scala-steward's PRs
+    conditions:
+      - author=scala-steward
+      - status-success=Travis CI - Pull Request
+    actions:
+      merge:
+        method: merge
```

Comment:
I know we haven't set this up here, but I far prefer squash merge, although I am not sure how much of a difference it makes

Reply:
Yeah i prefer squash too but i think steward always commits on recent master and rebases open PRs. So this should be the same i think. Well except maybe if I push a commit into stewards PR that makes it succeed. It’s a good point.